### ISSUE TITLE
Could you please provide the source code of fine-tuning LLaMA-2-7B-chat?

### ISSUE THREAD

**Issue by vhientran** [luciusssss/ZhuangBench#1]
Hi @luciusssss 

Many thanks for releasing the source code of your great paper. In Table 1 of your paper, you reported the performance of the LLaMA-2-7b-chat model when fine-tuning it. If possible, Could you please help to release this source code, as I am a beginner for fine-tuning LLMs.
Thank you very much for your consideration and your help!

**Comment by luciusssss** [comment]
Hi! We use Deepspeed (https://github.com/microsoft/DeepSpeed) to finetune LLaMA. You can access the code for training and inference from their Github repo.

We use the instruction "Please translate the following Chinese sentence into Zhuang:" in the prompt. The batch size is 64. The learning rate is 2e-5.

**Comment by vhientran** [comment]
I will try it. Thank you for your quick reply!
